script/DeployLocalDevNet.s.sol

```diff
@@ -86,6 +86,9 @@ contract DeployLocalDevNet is DeployBase {
         config.sealDuration = 0;
         config.sealExpiryTimestamp = 0;
 
+        // An address from the shared wallet, look for `shared-wallet.ts` in `lido-local-devnet`.
+        config.secondAdminAddress = 0xafF0CA253b97e54440965855cec0A8a2E2399896;
```

Comment:
There might be an error here because the wallet can change. It's better to use the `getNamedWallet` method on the local-devnet side, and use an environment variable here.